addon/components/new-bourbon-select-field.js

```diff
@@ -24,11 +23,10 @@ export default Component.extend({
   optionValuePath: null,
   optionEnabledPath: null,
   action: null,
-
+  lazyLoad: false,
```

Comment:
I don't think we need to add this anymore.

Reply:
deleting